Add context

dannyjacobs opened the pull request:
fixes "app context" issues introduced by upgrading to latest flask-sqlalchemy. seems to be working on site.

Commit:
fix error import error

Commit:
add useful deletion log output

Commit:
fix semicolon bug

Commit:
semicolon bug and store page fails gracefully when pots are down

Commit:
fix semicolon bug

Commit:
add context to M&C integration

Commit:
add context

bhazelton commented:
It looks like you added a number of changes that were not what we did today. I think this is maybe not based on main...

bhazelton commented:
I see, it's based on a branch called "updates_from_site". So merging this PR would merge both branches. Which is presumably ok.

plaplant committed:
Try removing a context manager

plaplant committed:
Add context managers to the wrapup functions

plaplant committed:
Add code after reformatting

bhazelton committed:
fix variable name in error message

bhazelton committed:
Move app_context up to include imports from file.py

importing file.py needs to be in a context because it has top level functions with app.route decorators.

bhazelton committed:
try to fix missing observation record error

bhazelton committed:
undo app context changes in mc_integration

Commit:
fix context issues, add more logging, increase m&c rate x60
